### PR TITLE
Added DELETE method for deleting an episode to the apiv2.

### PR DESCRIPTION
**Pull request by p0psicles**
DELETE: /series/slug/episodes/slug/

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by OmgImAlexis** [comment]
@p0psicles does this allow deleting the file associated?

**Comment by p0psicles** [comment]
@OmgImAlexis  No my change doesn't

**Comment by p0psicles** [comment]
@ratoaq2 I could use some help here with the api-description and the dredd tests?

File: dredd/api-description.yml [hunk 1]
```diff
@@ -2342,6 +2370,13 @@ parameters:
     description: The episode id to retrieve. E.g. s02e03, e34 or 2016-12-31
     x-example: s01e01
     type: string
+  episode-id-delete:
+    name: episode-id
```
Comment (OmgImAlexis) [inline dredd/api-description.yml]: Does this also need `-delete`?

Comment (OmgImAlexis) [reply dredd/api-description.yml]: What I mean is the ones above all have it matching with their name where as this one is missing the `-delete`. Might be related.

Comment (p0psicles) [reply dredd/api-description.yml]: Oh like that, i dont think the name makes any difference.

**Comment by p0psicles** [comment]
Dredd is fixed. Just need to update some unit tests

**Pull request merged**